docker/run.sh

```diff
@@ -3,10 +3,14 @@
 : "${GF_PATHS_DATA:=/var/lib/grafana}"
 : "${GF_PATHS_LOGS:=/var/log/grafana}"
 : "${GF_PATHS_PLUGINS:=/var/lib/grafana/plugins}"
+: "${GF_DASHBOARDS_ENABLED:=true}"
```

Comment:
Maybe we should keep "false" as the default, and override to true when wanted? `docker run -e "GF_DASHBOARDS_ENABLED=true" -i -p 3000:3000 (etc.)`